### PR TITLE
Fix invalid references to options in fixtures

### LINKED ISSUE #4411: [Fixtures][Install] OutOfBoundsException while installing fixtures
After recent changes on *Syliius* master, there is (again) problem with loading sample data.
zrzut ekranu 2016-03-07 o 20 49 57

I think it's next reason, why we should definitely implement Sylius installation tests as soon as possible ;)

### PR DESCRIPTION
**Pull request by pjedrzejewski**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | fixes #4411 
| License         | MIT
| Doc PR          | -

### CODE REVIEW

**Comment by okwinza** [comment]
:+1:

**Comment by pjedrzejewski** [comment]
We should generally favor ``code`` over ``name`` in fixtures, but that's separate issue. :)

File: src/Sylius/Bundle/FixturesBundle/DataFixtures/ORM/LoadProductOptionData.php [hunk 1]
```diff
@@ -127,7 +127,7 @@ protected function createOption($optionCode, $name, array $nameTranslation, arra
             $option->addValue($optionValue);
         }
 
-        $this->setReference('Sylius.Option.'.$name, $option);
+        $this->setReference('Sylius.Option.'.$optionCode, $option);
```
Comment (liverbool) [inline src/Sylius/Bundle/FixturesBundle/DataFixtures/ORM/LoadProductOptionData.php]: No need `$name` in 2nd function's parameter.

**Comment by michalmarcinkowski** [comment]
:+1:

**Pull request merged**